### PR TITLE
Alpine 3.18 new

### PR DESCRIPTION
**Pull request by kannon92**
Upgrade alpine to 3.18.

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-645) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by richscott — COMMENTED** [review]
I had just one small question on the first file change, otherwise it looks good.

File: .goreleaser.yml [hunk 1]
```diff
@@ -226,8 +226,7 @@ dockers:
       - --label=org.opencontainers.image.version={{ .Version }}
       - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
       - --label=org.opencontainers.image.revision={{ .FullCommit }}
-      - --label=org.opencontainers.image.base.name=alpine:3.17.0
-      - --label=org.opencontainers.image.base.digest=c0d488a800e4
+      - --label=org.opencontainers.image.base.name=alpine:3.18.3
```
[the remark this answers is not in the document]
Comment (kannon92) [reply .goreleaser.yml:229]: Dejan said we could remove in a slack thread.

[the remark this answers is not in the document]
Comment (richscott) [reply .goreleaser.yml:229]: 👍🏼

[the remark this answers is not in the document]
Comment (dejanzele) [reply .goreleaser.yml:229]: we don't maintain it and it gives no special interesting information

**Review by dejanzele — COMMENTED** [review]
LGTM

**Pull request merged**